src/search.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs');
+
+fs.readFile('./dict.txt', function(err, data){
+  if (err){
+    console.log(err);
+    return;
+  }
+  else{
+    var dict = (data.toString()).toLowerCase().split("\n");
+    // dict = dict.split("\n")
+    console.log(dict);
+  }
+})
+
+function search(userInput){
```

Comment:
Add a while loop so it stops as soon as you found five?

Or a counter that adds 1 as while loops can be dodgy